README.md

```diff
@@ -1,4 +1,4 @@
-# Nomad Autoscaler [![Build Status](https://circleci.com/gh/hashicorp/nomad-autoscaler.svg?style=svg)](https://circleci.com/gh/hashicorp/nomad-autoscaler) [![Discuss](https://img.shields.io/badge/discuss-nomad-00BC7F?style=flat)](https://discuss.hashicorp.com/c/nomad)
+# Nomad Autoscaler [![Discuss](https://img.shields.io/badge/discuss-nomad-00BC7F?style=flat)](https://discuss.hashicorp.com/c/nomad)
```

Comment:
```suggestion
# Nomad Autoscaler [![CI](https://github.com/hashicorp/nomad-autoscaler/actions/workflows/ci.yaml/badge.svg)](https://github.com/hashicorp/nomad-autoscaler/actions/workflows/ci.yaml) [![Discuss](https://img.shields.io/badge/discuss-nomad-00BC7F?style=flat)](https://discuss.hashicorp.com/c/nomad)
```

This gives a GHA badge, which is what we've been using now 

[![CI](https://github.com/hashicorp/nomad-autoscaler/actions/workflows/ci.yaml/badge.svg)](https://github.com/hashicorp/nomad-autoscaler/actions/workflows/ci.yaml)